doc/classes/Physics2DServer.xml

```diff
@@ -1028,14 +1128,23 @@
 			<argument index="1" name="data" type="Variant">
 			</argument>
 			<description>
-				Sets the shape data that defines its shape and size. The data to be passed depends on the kind of shape created [method shape_get_type].
+				Sets the shape data that defines its shape and size. The data to be passed depends on the type of shape created (see. [method shape_get_type]). The following are the Variants required for each [enum ShapeType].
+				[b]SHAPE_LINE[/b]: [method shape_set_data] [codeblock]( RID shape, array[Vector2() , float length] )[/codeblock]
+				[b]SHAPE_RAY[/b]: [method shape_set_data] [codeblock]( RID shape, {”length”: float length, “slips_on_slope”: bool} )[/codeblock]
+				[b]SHAPE_SEGMENT[/b]: [method shape_set_data] [codeblock]( RID shape,array[float pointA.x, float pointA.y, float pointB.x, float pointB.y] )[/codeblock]
+				[b]SHAPE_CIRCLE[/b]: [method shape_set_data] [codeblock]( RID shape, float radius)[/codeblock]
+				[b]SHAPE_RECTANGLE[/b]: [method shape_set_data] [codeblock]( RID shape, array[float width, float height)] )[/codeblock]
+				[b]SHAPE_CAPSULE[/b]: [method shape_set_data] [codeblock]( RID shape, array[float height, float width] )[/codeblock]
+				[b]SHAPE_CONVEX_POLYGON[/b]: [method shape_set_data] [codeblock]( RID shape, array[Vector2(), Vector2(), ...] )[/codeblock]
+				[b]SHAPE_CONCAVE_POLYGON[/b]: [method shape_set_data] [codeblock]( RID shape, array[Vector2(), Vector2(), ...] )[/codeblock]
+				Note: SHAPE_CONVEX_POLYGON is the same type used by CollisionPolygon2D when it is set to “solids,” and SHAPE_CONCAVE_POLYGON  is used when CollisionPolygon2D is set to “segments.”
```

Comment:
Those `SHAPE_*` strings should be references to the appropriate constants. Also, inline code is formatted with `[code]` not `[codeblock]`.